### PR TITLE
MSR: A5 lobby

### PR DESCRIPTION
**Pull request by Miepee**

### CODE REVIEW

File: randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt [hunk 1]
```diff
@@ -195,8 +197,10 @@ Extra - asset_id: collision_camera_004
           Any of the following:
               # Reaching the ledge
               Single-wall Wall Jump (Beginner) or Climb Rooms Vertically (No High Jump)
-              # Freeze the Gullugg
+              # Freeze the Gullugg - TODO: get rid of template
               Stand on Frozen Enemy (Beginner) and Wall Jump (Beginner) and Fully Freeze Enemy
+              # Get up with HJ
+              High Jump Boots or Super Jump (Intermediate)
```
Comment (dyceron) [inline randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt:203]: This path is wrong but impossible anyway. Remove

File: randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt [hunk 2]
```diff
@@ -893,15 +910,11 @@ Extra - asset_id: collision_camera_014
   > Pickup (Phase Drift)
       Any of the following:
           Morph Ball
-          # https://youtu.be/P18XiY9K36o
+          # Corner Clip: https://youtu.be/P18XiY9K36o
           Corner Clip (Intermediate) and Out of Bounds Movement (Intermediate)
```
Comment (dyceron) [inline randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt:914]: We want to delete the Corner Clip trick entirely, so make this just oob movement (and delete the Trick from the header)

**Review by dyceron — CHANGES_REQUESTED** [review]

File: randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt (hunk 1 above)
Comment (Miepee) [reply randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt:203]: Oh, I added it to the wrong connection, whoops

File: randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt (hunk 2 above)
Comment (Miepee) [reply randovania/games/samus_returns/logic_database/Area 5 - Tower Lobby.txt:914]: will delete from the header at a later date to ensure less conflicts

**Review by dyceron — CHANGES_REQUESTED** [review]
There's so a Trivial connection to phase Drift pickup b fun the dock which is wrong. Just remove that connection

**Comment by Miepee** [comment]
whops, done

**Review by dyceron — APPROVED** [review]

**Pull request merged**